### PR TITLE
add new method: listEndNodes

### PR DESCRIPTION
**Pull request by knqiufan**
add new method: listEndNodes
add new test case about listEndNodes

### CODE REVIEW

**Review by CorvusYe — APPROVED** [review]
There's not much of a problem with the order. Thanks for your time.

This is for #261.
cc: @wey-gu

File: ngbatis-demo/src/test/java/ye/weicheng/ngbatis/demo/NebulaBasicDaoTests.java [hunk 1]
```diff
@@ -501,6 +501,12 @@ public void listStartNodes() {
     System.out.println(JSON.toJSONString(personList));
   }
 
+  @Test
+  public void listEndNodes() {
+    List<Person> personList = repository.listEndNodes("易小海", Like.class);
```
Comment (shbone) [inline ngbatis-demo/src/test/java/ye/weicheng/ngbatis/demo/NebulaBasicDaoTests.java:506]: Shoud we default think that the user know `endType` of endNodes? @CorvusYe

File: src/main/java/org/nebula/contrib/ngbatis/proxy/NebulaDaoBasic.java [hunk 2]
```diff
@@ -440,6 +452,25 @@ default List<?> listStartNodes(Class<?> startType, Class<?> edgeType, I endId) {
...
 
+  /**
+   * 指定结束节点类型 并通过开始节点id与关系类型获取所有结束节点
+   *
+   * @param endType 结束接待你的类型
+   * @param edgeType 关系类型
+   * @param startId 开始节点的id
+   * @return 结束节点列表
+   */
+  default List<?> listEndNodes(Class<?> endType, Class<?> edgeType, I startId) {
```
[the remark this answers is not in the document]
Comment (shbone) [reply src/main/java/org/nebula/contrib/ngbatis/proxy/NebulaDaoBasic.java]: That's great update!

**Comment by shbone** [comment]
> @shbone Do you have any suggestions for this implementation?
I have no suggestion

**Comment by shbone** [comment]
> @shbone Do you have any suggestions for this implementation?

NO, I have no suggestion temporarily

File: ngbatis-demo/src/test/java/ye/weicheng/ngbatis/demo/NebulaBasicDaoTests.java (hunk 1 above)
Comment (CorvusYe) [reply ngbatis-demo/src/test/java/ye/weicheng/ngbatis/demo/NebulaBasicDaoTests.java:506]: Like listStartNodes, listEndNodes provides two apis where you can specify the type of endNode. 
Do I understand your question correctly?

Comment (CorvusYe) [reply ngbatis-demo/src/test/java/ye/weicheng/ngbatis/demo/NebulaBasicDaoTests.java:506]: > Shoud we default think that the user know `endType` of endNodes? @CorvusYe 

@shbone 
Like listStartNodes, listEndNodes provides two apis where you can specify the type of endNode. 
Do I understand your question correctly?

Comment (shbone) [reply ngbatis-demo/src/test/java/ye/weicheng/ngbatis/demo/NebulaBasicDaoTests.java:506]: Yes，one is that you can specify the type of endNode, another is that you dont't know the type of endNode tha you  just know the type of startNode and edge.

**Pull request merged**